org/SimAces/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: org
+title: Brian Todoroff
```

Comment:
Either this should be "Sim Aces", or your orgname (the directory this file is in) should be your own name.